### ISSUE TITLE
Play mp4 videos

### ISSUE THREAD

**Issue by unoiatech** [Viblast/ios-player-sdk#4]
Can we play live streaming videos or mp4 ?

**Comment by ivermolaev** [comment]
Yes of course. As long as the live stream is transported via DASH you should just provide its URL to `-initWithCDN:`. You should basically do the same if you have a .mp4 hosted somewhere.
